homework2/pizza_order.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Pizza Order</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <nav>
+        <div class="menu">
+            <img class="img" src="pizza.png" width="70px" height="50px">
+            <a href="#">Home</a>
+            <a href="#">About Us</a>
+            <a href="#">Our Clients</a>
+            <a href="#">Contact Us</a>
+        </div>
+    </nav>
+
+    <form method="post">
+        <h3>Pizza Order</h3>
+        <label for="name">Your name:</label>
+        <input type="text" id="name" name="name" minlength="3" maxlength="10" required>
+
+        <br><br>
+
+        <label for="emailInput">Your email:</label>
+        <input type="text" id="emailInput" name="email" required>
+
+        <br><br>
+
+        <label for="quantity">Quantity:</label>
+        <select name="quantity">
+            <option value="1">1</option>
```

Comment:
Pentru fiecare option, ar trebui sa avem un value atribute diferit.
valoarea pe care o punem in value o sa fie trimisa catre back-end pentru a salva informatia. Aici, daca ai pus peste tot valoarea 1, asta inseamna ca, indiferent de ce alege utilizatorul, nu vom sti decat ca a ales optiunea 1. Astfel, celelalte optiuni vor fi ignorate.